cves/CVE-2014-0050.yml

```diff
@@ -204,4 +223,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I would say that this would be a coding mistake. In the implementation of this piece of code, the author had mentioned
+    in the Git log that he had attempted to account for all of the "sneaky boundary cases". This goes to show just how 
+    sneaky they can be in that there was a bug stemming from input being above a certain length. 
+
+    That said, it is difficult to blame this on the programmer, given that the vulnerability could only be exploited by an attack 
+    vector specifically meant to be strange and unlikely.
```

Comment:
Retrospectively, is this something the programmer _should_ have accounted for? If not, how else could this vulnerability have been prevented?